tests/test_libraries.py

```diff
@@ -2660,3 +2660,10 @@ def test_niquests(pyi_builder):
         except (niquests.ConnectionError, niquests.Timeout):
             pass
     """)
+
+
+@importorskip('emoji')
+def test_vadersentiment(pyi_builder):
```

Comment:
This should be `test_emoji`, which is what linter is complaining about.

Reply:
Fixed.